### PR TITLE
fix(actions) : update get-stage and setup-yarn actions to remove set-output warning

### PR DESCRIPTION
**Pull request by ridwan-b**
- update @actions/core from 1.2.5 to 1.10.0
- use  $GITHUB_OUTPUT as recommanded by github documentation

### CODE REVIEW

**Review by vmaerten — COMMENTED** [review]
for the type, it's more a refactor or a fix no ?
for the scope, as we are in the "actions" repo I assume, it's about the actions :trollface: 

Did you test it in another repo ?

otherwise LGTM !

**Comment by tmuzy** [comment]
the bug label was just a test with labels @ridwan-b

**Review by cforycki — APPROVED** [review]
lgtm, but I'm no expert on actions

**Comment by ridwan-b** [comment]
@vmaerten the master branch is currently in used into kleanads-prebid repo. The two stages setup-yarn and get-stage are working well

**Comment by vmaerten** [comment]
> @vmaerten the master branch is currently in used into kleanads-prebid repo. The two stages setup-yarn and get-stage are working well

But the master branch does not contain your modification right?

File: get-stage/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "license": "MIT",
   "homepage": "https://github.com/pbstck/actions#README",
   "dependencies": {
-    "@actions/core": "^1.2.5",
+    "@actions/core": "^1.10.0",
     "@actions/github": "^1.1.0"
```
Comment (cforycki) [inline get-stage/package.json:21]: By the way, should we update this aswel? Latest is `5.1.1`

**Review by sdeluce — APPROVED** [review]

File: get-stage/package.json (hunk 1 above)
Comment (cforycki) [reply get-stage/package.json:21]: My bad, it's on @actions/github

**Comment by vmaerten** [comment]
We merge it because we need it :)

**Pull request merged**